matrix_sdk_common/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-authors = ["Damir Jelić <poljar@termina.org.uk>"]
+authors = ["Damir Jelić <poljar@termina.org.uk"]
```

Comment:
Let's not mess up the authors again.

Reply:
Perfect this gives me a slightly less awkward way to ask if it would be appropriate for me to add my name to `matrix_sdk_base` and maybe `matrix_sdk`?

Reply:
We want  to use the `authors` field for the maintainer, not  the original authors. This should eventually end up being a generic Matrix.org Foundation contact. I know this isn't the current state of affairs but that's the plan.

Authorship of the source code is tracked in the copyright headers, if you feel like we need more visibility for the contributors we can add a `CONTRIBUTORS.md` file.

Reply:
Ah I see, no worries! I was curious about it and kinda figured it would be a company thing.